src/Service/MagicLinks/Models/SendMagicLinkRequest.cs

```diff
@@ -0,0 +1,30 @@
+using System.ComponentModel.DataAnnotations;
+using System.Net.Mail;
+using Passwordless.Service.MagicLinks.Validation;
+
+namespace Passwordless.Service.MagicLinks.Models;
+
+public class SendMagicLinkRequest
+{
+    [Required]
+    [EmailAddress]
+    public string EmailAddress { get; init; }
+
+    [Required]
+    [MagicLinkTemplateUrl]
+    public string LinkTemplate { get; init; }
+
+    [Required]
+    public string UserId { get; init; }
+}
+
+public static class SendMagicLinkRequestExtensions
+{
+    public static MagicLinkDTO ToDto(this SendMagicLinkRequest request) =>
```

Comment:
Curious, why not just an instance method?

Reply:
Force of habit. I've always just written extension methods when mapping between two object.s

Reply:
Seems like other models use instance methods in this PR, should we stay consistent with that? 🙂